content/src/content/jcr_root/apps/core/wcm/components/carousel/v1/carousel/carousel.html

```diff
@@ -15,14 +15,19 @@
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
 <div data-sly-use.carousel="com.adobe.cq.wcm.core.components.models.Carousel"
      data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
+     aria-label="${carousel.carouselLabel}"
+     aria-roledescription="carousel"
      class="cmp-carousel"
+     role="group"
```

Comment:
Would list the accessibility attributes after the css class, with the role first, something like:
```
class="cmp-carousel"
role="group"
aria-roledescription="carousel"
aria-label="${carousel.carouselLabel}"
```

Reply:
I can see that buttons have those in this order: role -> class -> other stuff. Would you want me to change the buttons to have those attributes in the order you mentioned, or maybe make other things have the same order of attributes as those buttons (role before class)?

Reply:
We have different orderings per component right now and need a clean up task to make sure we're consistent, but I think that order I posted makes the most sense (they're the same order you would see when inspecting an element in the console). We should document it in our HTL guidelines. In full, I think it should be:

```
HTL directives
CSS classes
Role attribute
Aria attributes
Data attributes
```

As a first step it makes sense to make sure we're consistent in this file, then later take on the tidy up for other components.

Reply:
Sounds good, changed the order of attributes to the order you mentioned